crates/bevy_winit/src/converters.rs

```diff
@@ -274,3 +274,17 @@ pub fn convert_window_level(window_level: WindowLevel) -> winit::window::WindowL
         WindowLevel::AlwaysOnTop => winit::window::WindowLevel::AlwaysOnTop,
     }
 }
+
+pub fn convert_winit_theme(theme: winit::window::Theme) -> WindowTheme {
+    match theme {
+        winit::window::Theme::Light => WindowTheme::Light,
+        winit::window::Theme::Dark => WindowTheme::Dark,
+    }
+}
+
+pub fn convert_window_theme(theme: WindowTheme) -> winit::window::Theme {
+    match theme {
+        WindowTheme::Light => winit::window::Theme::Light,
+        WindowTheme::Dark => winit::window::Theme::Dark,
+    }
+}
```

Comment:
Any reason for not implementing `From` for these types, and instead writing these converter functions? I'm probably ignoring a bunch of stuff so please bear with me. 😄

Reply:
You can implement `From<X> for Y` in the crate that defines `X`. Trivially, `Color` implements `From<Vec4>`, yet `Vec4` is defined in the glam crate. This is also true of the opposite: `From<Color> for Vec4`.

See:
- [`From<Vec4> for Color`](https://docs.rs/bevy/latest/bevy/prelude/enum.Color.html#impl-From%3CVec4%3E-for-Color)
- [`From<Color> for Vec4`](https://docs.rs/bevy/latest/bevy/prelude/enum.Color.html#impl-From%3CColor%3E-for-Vec4)